src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -0,0 +1,154 @@
+foam.CLASS({
+  package: 'foam.u2.dialog',
+  name: 'ApplicationPopup',
+  extends: 'foam.u2.dialog.Popup',
+  documentation: `
+    A full-featured popup with the application's branding on it.
+  `,
+
+  implements: ['foam.mlang.Expressions'],
+
+  imports: [
+    'theme'
+  ],
+
+  exports: [
+    'close as closeDialog',
+    'as popup'
+  ],
+
+  requires: [
+    'foam.u2.ActionReference',
+    'foam.u2.tag.Image'
+  ],
+
+  css: `
+    ^X {
+      margin: 15px;
+      position: initial !important;
```

Comment:
Why the importants? Why not make a new class if we are going to override everything

Reply:
A new class wouldn't cut it - it would have to be a CSS selector with higher specificity too. Maybe `^X^ApplicationPopupX`. I don't think using `important` is a problem here though; the real problem is that CSS is really useless within some mechanism for inheritance.

Reply:
Well you override the render in this class anyway so you can get rid of the X CSS class anyway